<launch>

  <node pkg="floor_plane_mapping" type="floor_plane_mapping" name="floor_plane_mapping" output="screen">
      launch-prefix="xterm -e gdb --args"
      <param name="base_frame" value="/bubbleRob" />
      <param name="max_range" value="0.5" />
      <param name="traverse_threshold" value="0.6" />
      <param name="normal_estimation_radius" value="0.03" />

      <remap from="/floor_plane_mapping/scans" to="/vrep/depthSensor"/>
  </node>
</launch>
